mon-pix/translations/en.json

```
@@ -564,25 +564,25 @@
                 "message": "<h2>Hi {firstname}, welcome to your dashboard.</h2><p>Quickly and easily access all your tests in progress.<br>Follow the evolution of your Pix score and check if your profile is ready for certification.</p>"
             },
             "recommended-competences": {
-                "profile-link": "See all skills",
-                "subtitle": "Discover the skills recommended for you.",
-                "title": "Recommended skills"
+                "profile-link": "See all competences",
+                "subtitle": "Discover the competences recommended for you.",
+                "title": "Recommended"
             },
             "improvable-competences": {
                 "profile-link": "See all",
                 "subtitle": "Ready to improve your score?",
                 "title": "Try again"
             },
             "score": {
-                "profile-link": "See my skills"
+                "profile-link": "See my competences"
             },
             "started-competences": {
-                "subtitle": "Continue your tests, find the most recent ones here.",
-                "title": "Resume testing a skill"
+                "subtitle": "Resume testing your skills, find your most recent tests here.",
+                "title": "In progress"
             }
         },
         "error": {
-            "content-text": "'<p>'Please refresh the page or '<LinkTo @route=\"login\">'return to homepage'</LinkTo>'.'</p><p>'You can also contact us via '<a href=\"https://support.pix.fr/support/tickets/new\">'the help centre form'</a>'specifying the error code below in the description.'</p><p>'Please excuse any inconvenience caused.'</p><p>'The Pix team.'</p>'",
+            "content-text": "'<p>'Please refresh the page or '<LinkTo @route=\"login\">'return to homepage'</LinkTo>'.'</p><p>'You can also contact us via '<a href=\"https://pix.org/en-gb/help-form\">'the help centre form'</a>'specifying the error code below in the description.'</p><p>'Please excuse any inconvenience caused.'</p><p>'The Pix team.'</p>'",
```

```suggestion
            "content-text": "'<p>'Please refresh the page or '<LinkTo @route=\"login\">'return to homepage'</LinkTo>'.'</p><p>'You can also contact us via '<a href=\"https://pix.org/en-gb/faq-pix\">'the help centre form'</a>'specifying the error code below in the description.'</p><p>'Please excuse any inconvenience caused.'</p><p>'The Pix team.'</p>'",
```
Sinon l'url n'existe pas :s

---

Cet url `https://pix.org/en-gb/help-form` fonctionnait avant, on a plusieurs références sur ce lien, je vois avec la team traduction

---

il faut remplacer toutes les occurrences par `https://pix.org/en-gb/contact-form`